### ISSUE TITLE
passing filenames from deoplete to gocode under cygwin (windows)

### ISSUE THREAD

**Issue by yadaniel** [Shougo/deoplete.nvim#832]
gocode installed in
/cygdrive/d/prog/go/_GOPATH/src/deoplete/main.go
and is tested with separate fmt.go (as found https://github.com/zchee/deoplete-go/issues/50)
the result is correct
[7,[{"class":"func","package":"fmt","name":"Println","type":"func(a ...interface{}) (n int, err error)"}]]

vim plugins installed
Plugin 'Shougo/deoplete.nvim'
Plugin 'roxma/nvim-yarp'
Plugin 'roxma/vim-hug-neovim-rpc'
Plugin 'zchee/deoplete-go', { 'do': 'make'}

:echo has("python3") => 1

vim is giving the following error message
[deoplete] NoneType: None^@gocode panicked.  Use :messages / see above for error details.^@

$ gocode -s debug
panic: open D:\cygdrive\d\prog\go\_GOPATH\src\deoplete\: Das System kann den angegebenen Pfad nicht finden.

in summary I think that the problem is how the filename is passed from deoplete to gocode
it should be
D:\prog\go\_GOPATH\src\deoplete\
instead of
D:\cygdrive\d\prog\go\_GOPATH\src\deoplete\
